tests/shared/src/main/scala/munit/scalacheck/ScalaCheckFrameworkSuite.scala

```diff
@@ -0,0 +1,34 @@
+package munit
+package scalacheck
+
+import org.scalacheck.Prop.forAll
+import org.scalacheck.rng.Seed
+
+class ScalaCheckFrameworkSuite extends ScalaCheckSuite {
+
+  override def scalaCheckTestParameters =
+    super.scalaCheckTestParameters.withInitialSeed(Seed(123L))
+
+  property("list concatenation") {
+    forAll { (l1: List[Int], l2: List[Int]) =>
+      l1.size + l2.size == (l1 ::: l2).size
+    }
+  }
+
+  property("squared") {
+    forAll { (n: Int) =>
+      scala.math.sqrt(n * n) == n
+    }
+  }
+}
+
+object ScalaCheckFrameworkSuite
+    extends FrameworkTest(
+      classOf[ScalaCheckFrameworkSuite],
+      s"""|==> success munit.scalacheck.ScalaCheckFrameworkSuite.list concatenation
+          |==> failure munit.scalacheck.ScalaCheckFrameworkSuite.squared -${' '}
+          |Falsified after 0 passed tests.
+          |> ARG_0: -1
+          |> ARG_0_ORIGINAL: 2147483647
+          |""".stripMargin
+    )
```

Comment:
@olafurpg do you know why I don't see the exception name in the test (it should be where the `' '` is.

Also, the `println` output isn't captured here (but it appears when running `FrameworkSuite`)

Reply:
That's expected, `FrameworkSuite` renders custom output based on the published sbt events. This is the most reliable way I have seen so far to avoid flaky failures. There's a flag to include stdout

https://github.com/scalameta/munit/blob/37a04596254d972ee4012aa3ce95601b27fa22a4/tests/shared/src/main/scala/munit/FixtureFrameworkSuite.scala#L80

Reply:
👍 thanks for the explanation